tests/antithesis/Makefile

```diff
@@ -41,5 +44,38 @@ antithesis-run-local-validation:
 
 .PHONY: antithesis-clean
 antithesis-clean:
-	export USER_ID=$(USER_ID) && export GROUP_ID=$(GROUP_ID) && docker-compose down
+	export USER_ID=$(USER_ID); export GROUP_ID=$(GROUP_ID); docker-compose down
 	rm -rf /tmp/etcddata0 /tmp/etcddata1 /tmp/etcddata2 /tmp/etcdreport
+
+.PHONY: set-version-3.4
+set-version-3.4:
+	$(eval VERSION=3.4)
+	$(eval DOCKERFILE=./Dockerfile-release)
+
+.PHONY: antithesis-build-etcd-image-3.4
+antithesis-build-etcd-image-3.4: set-version-3.4 antithesis-build-etcd-image
+
+.PHONY: antithesis-docker-compose-up-3.4
+antithesis-docker-compose-up-3.4: antithesis-build-etcd-image-3.4 antithesis-docker-compose-up
+
+.PHONY: set-version-3.5
+set-version-3.5:
```

Comment:
Nice, I didn't know you can do this. Maybe we could use this approach to cleanup https://github.com/etcd-io/etcd/blob/55ee50089f9fa2f963a06110cf77a1d27347c40e/tests/robustness/Makefile#L103-L197